Fluid.Tests/ColorFiltersTests.cs

```diff
@@ -36,6 +37,27 @@ public void ToRgb(string color, string expected)
             Assert.Equal(expected, result.Result.ToStringValue());
         }
 
+        [Theory]
+        [InlineData("en-US")]
+        [InlineData("de")]
+        [InlineData("fr-FR")]
+        [InlineData("zh-Hans")]
+        public void ToRgbShouldNotBeAffectedByCurrentCulture(string culture)
+        {
+            // Arrange
+            CultureInfo.CurrentCulture = CultureInfo.GetCultureInfo(culture);
```

Comment:
Create private method named `SetCurrentCulture()` instead of repeating the code over and over